options.go

```diff
@@ -157,19 +163,23 @@ func DefaultOptions(path string) Options {
 		LogRotatesToFlush:             2,
 		EncryptionKey:                 []byte{},
 		EncryptionKeyRotationDuration: 10 * 24 * time.Hour, // Default 10 days.
+		KeepBlocksInCache:             false,
+		KeepBlockIndicesInCache:       false,
```

Comment:
Is it the same cache that is referenced in https://github.com/dgraph-io/badger/commit/21735af853dab8942e187dddbeeb41f3ecb04071?